backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/proto/GetTokenInfoResponse.java

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2020 Google LLC
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package com.google.gpay.virtualqueue.backendservice.proto;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+import com.google.gpay.virtualqueue.backendservice.model.Token;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+public class GetTokenInfoResponse {
+    private AtomicInteger customersAhead;
```

Comment:
Same here, why do we want to use AtomicInteger? Can we use long instead?

Reply:
yes done